### DISCUSSION TITLE
domain redirect doesn't work.

### DISCUSSION THREAD

**Discussion by ihorjs** [vercel/vercel#4858] [Help] [answered]
I have domain.com and www.domain.com
I want domain.com to redirect to www.domain.com to boost SEO.

I followed this blog and it didn't work.
https://vercel.com/blog/redirecting-domains.

What else do I have to do?
Am I missing else?

Please help me.
Thanks.

**Comment by dfjs** [comment, marked as answer]
@ihorjs I raised this same issue with Vercel support yesterday (around 15 Jul 2020, 10:40 BST) - since then it seems the issue is resolved / redirects are working again, though no response from the support team yet. 

FYI I observed this issue happening across multiple accounts and domains.
